packages/react-core/src/next/components/Select/examples/SelectTypeahead.tsx

```diff
@@ -26,6 +26,7 @@ export const SelectBasic: React.FunctionComponent = () => {
   const [filterValue, setFilterValue] = React.useState<string>('');
```

Comment:
@kmcfaul It doesn't seem like we need both `filterValue` and `inputValue` in this example. Was there a use case for this when the example was originally written?

Reply:
Hm yeah I think I was just separating them logically but it's unneeded in this example. It would be useful in the case where the filtering should be done after the full search term is typed in with a search button, rather than filtering as a user types (which is what this example shows).

Reply:
We can remove one or the other